test/test_torch.py

```diff
@@ -7734,6 +7734,20 @@ def test_helper(dim1, dim2, memory_format):
         test_helper((3, 3), (3, 3, 3, 3), torch.channels_last)
         test_helper((3, 3, 3), (3, 3, 3, 3, 3), torch.channels_last_3d)
 
+    def test_dim_order(self):
+        shape = (2, 3, 5, 7)
+
+        self.assertSequenceEqual(torch.empty(shape).dim_order(), (0, 1, 2, 3), seq_type=tuple)
+        self.assertSequenceEqual(torch.empty(shape).transpose(0, 1).dim_order(), (1, 0, 2, 3))
```

Comment:
Dont know, but this one might be worth explaining that transpose permutes those two dims. Thus if you have `t.sizes() = [2, 3, 4, 5]` then `t.tranpose(0, 1).sizes() = [3, 2, 4, 5]` even though underlying memory has not been shuffled. Which is why new dim order is (1, 0, 2, 3).

Reply:
Just adding a comment saying since this is stride based, expect dim_order to change with this test.